reno.yaml

```diff
@@ -0,0 +1,29 @@
+---
+collapse_pre_releases: false
+earliest_version: 0.104.0
+add_release_date: false
+stop_at_branch_base: true
+ignore_notes: []
+
+sections:
+  - [features, New Features]
+  - [fixes, Bug Fixes]
+  - [security, Security Issues]
+  - [issues, Known Issues]
+  - [upgrade, Upgrade Notes]
+  - [deprecations, Deprecations]
+  - [other, Other Notes]
+
+unreleased_version_title: ''
+
+prelude_section_name: release_summary
+template: |
+  ---
+  release_summary:
```

Comment:
`release_summary` should just be fillable by the maintainers right? Other contributors should just be able to add summary for other changes in other fields.

Reply:
Maybe. We can remove that from the template if you think it would be confusing.

Reply:
On the second thought, I don't think it's a good idea to change the template. Maintainers will most likely use the same command to create a note.

Reply:
hmm, My point is, release_summary is something that, if needed, should be part of entirely different PR (just the release_summary) and that PR would be raised once per release.
If every contributor gets that field, they might add things their, if we are ok to take that risk I am ok.

Reply:
We can remove it or ask to remove it at any point. There is a point during pre-release process when we're supposed to review the release notes.